### PR TITLE
Update branching docs

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A preview-label label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: Documentation/Branching.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+<h1 align="center">Branching</h1>
+
+**Velaptor** uses a more complicated branching model, but it gives you more control of the SDLC (Software Development Life Cycle).  This branching model allows a clear purpose for adding features, bug fixes, preview releases, QA releases and standard releases.
```
Comment (KinsonDigitalAdmin) [inline Documentation/Branching.md:3]: Update project name to **VersionMiner**

```suggestion
**VersionMiner** uses a more complicated branching model, but it gives you more control of the SDLC (Software Development Life Cycle).  This branching model allows a clear purpose for adding features, bug fixes, preview releases, QA releases and standard releases.
```

File: Documentation/Branching.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+**_NOTE_:** As you know, everything in software is subject to change, including the branching model.  If too many issues or complications occur with the current branching model and/or release process, it will be changed accordingly.
+
+---
+
+<h2 align="center">Branches Used</h2>
+
+
+<h3 align="left" style="color: #82B366;font-weight:bold">Master Branch</h3>
+
+Long living branch that represents stable production versions of **Velaptor**:
```
Comment (KinsonDigitalAdmin) [inline Documentation/Branching.md:16]: Update project name to **VersionMiner**

```suggestion
Long living branch that represents stable production versions of **VersionMiner**:
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]
